testsuite/MDAnalysisTests/analysis/test_nucleicacids.py

```diff
@@ -43,3 +44,27 @@ def test_wc_dist(u):
 
     assert_allclose(WC.results[0][0], 4.3874702, atol=1e-3)
     assert_allclose(WC.results[1][0], 4.1716404, atol=1e-3)
+
+
+def test_minor_dist(u):
+    strand: mda.AtomGroup = u.select_atoms("segid RNAA")
+    strand1 = [strand.residues[2], strand.residues[19]]
+    strand2 = [strand.residues[16], strand.residues[4]]
+
+    MI = MinorPairDist(strand1, strand2)
+    MI.run()
+
+    assert_allclose(MI.results[0][0], 15.06506, atol=1e-3)
```

Comment:
another one for @MDAnalysis/coredevs to consider for our testing style going ahead - single float comparisons are ~ 20x faster (at least on my laptop) if done using `pytest.approx` instead of `assert_allclose`. It's microseconds, but it adds up quickly to seconds, how do we feel about making pytest.approx the assert method of choice in those cases?

Reply:
I raised #3743 for you.

Reply:
I refactored my test to use pytest approx